newrelic/api/opentelemetry.py

```diff
@@ -296,6 +304,63 @@ def record_exception(self, exception, attributes=None, timestamp=None, escaped=F
 
         notice_error(error_args, attributes=attributes)
 
+    def _database_attribute_mapping(self):
```

Comment:
I like where you are going with this in terms of wrapping all the attributes up in some sort of mapper container. I think maybe we should roll with this concept but make it a bit more official. Can you move this outside of this class and make it a function that returns a dictionary map of the attributes? So something like:
```
def map_database_attributes(attributes):
   return {
       "host": attributes.get("net.peer.name") or attributes.get("server.address"),
       ...
   }
```